### DISCUSSION TITLE
Nouveaux status

### DISCUSSION THREAD

**Discussion by DarkThmo** [Crownicles/Crownicles#692] [Nouvelles idées]
Je propose une classe healeur, pour pouvoir aider les gens de sa guilde a retrouver du soin contre de l'argent,
Cela rajouterait une fonctionnalité de plus aux guildes je trouve donc voila.

**Comment by Thesmokingllama** [comment]
En effet ça pourrait être cool, contre de l'argent jsp si c'est une bonne idée mais avoir la possibilité de soigner régulièrement une personne est excellente ! Ça pourrait très bien aller dans une série de nouvelles classes à capacités comme le soin (mentionné ici) , le karma (évoqué par piet), de dressage  (que je me permets d'ajouter à la liste) etc...

**Reply by DarkThmo** [reply]
haha ^^

**Reply by BastLast** [reply]
Des nouvelles classes sont définitivement prévues, on fera en fonction des nouvelles features ajoutées pr que ca soit sympa !
